apps/ensadmin/src/app/name/[name]/page.tsx

```diff
@@ -0,0 +1,310 @@
+"use client";
+
+import { Avatar, AvatarFallback, AvatarImage } from "@/components/ui/avatar";
+
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+
+import { Skeleton } from "@/components/ui/skeleton";
+import { getNameAvatarUrl } from "@/lib/namespace-utils";
+import { ENSNamespaceIds } from "@ensnode/datasources";
+import { useRecords } from "@ensnode/ensnode-react";
+import { DefaultRecordsSelection } from "@ensnode/ensnode-sdk";
+import { ExternalLink, Mail, Twitter } from "lucide-react";
+import { useParams } from "next/navigation";
+import { useEffect, useState } from "react";
+
+export default function NameDetailPage() {
+  const params = useParams();
+  const name = decodeURIComponent(params.name as string);
+  const [mounted, setMounted] = useState(false);
+
+  useEffect(() => {
+    setMounted(true);
```

Comment:
I really dislike this, so I'm going to defer to using the ensnode-sdk so we can wrap this page with Suspense (`loading.tsx`).